nestkernel/sonata_connector.cpp

```diff
@@ -0,0 +1,667 @@
+/*
+ *  sonata_connector.cpp
+ *
+ *  This file is part of NEST.
+ *
+ *  Copyright (C) 2004 The NEST Initiative
+ *
+ *  NEST is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation, either version 2 of the License, or
+ *  (at your option) any later version.
+ *
+ *  NEST is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License
+ *  along with NEST.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+#include "sonata_connector.h"
+#include "config.h"
+
+#ifdef HAVE_HDF5
+
+#include <cstdlib> // for div()
+
+// Includes from nestkernel:
+#include "kernel_manager.h"
+#include "vp_manager_impl.h"
+
+// Includes from sli:
+#include "dictutils.h"
+
+#include <chrono>   // for debugging
```

Comment:
Can we use a debug flag from cmake here?

Reply:
I just removed the includes as they were not used anywhere.